os/arch/arm/src/tiva/lm3s_ethernet.c

```diff
@@ -848,9 +688,7 @@ static void tiva_receive(struct tiva_driver_s *priv)
 					} else
 #endif
 #ifdef CONFIG_NET_IPv6
```

Comment:
I believe this is already enclosed within a similar check at line 668. Could you please confirm?